build.sh

```diff
@@ -6,11 +6,13 @@
 # Dependencies include: build-essential libncurses5 libncurses-dev cmake zlib1g-dev
 set -e # exit on first error
 
-jobs=4
+jobs=$(echo $(nproc) || echo $(sysctl -n hw.physicalcpu))
```

Comment:
Can we make it one line rather than a if condition

Reply:
Would it be faster when you were trying to build svf using more than 4 threads? I think we can leave it as 4 and people can simply adjust it by themselves

Reply:
much faster when I use blade with higher jobs.